test/e2e_node/image_list.go

```diff
@@ -76,6 +76,10 @@ func updateImageAllowList() {
 	// Images from extra envs
 	framework.ImagePrePullList.Insert(getNodeProblemDetectorImage())
 	framework.ImagePrePullList.Insert(getSRIOVDevicePluginImage())
+
+	// Delete the empty string,
```

Comment:
why is there an empty string to begin with?

Reply:
https://github.com/kubernetes/kubernetes/blob/7e6ef0efb6555bbd3720ac97bf913e8093fe7289/test/e2e_node/image_list.go#L216-L232
It is returned here. 
In the case of a bad network, an empty string may be returned.

But this should not affect the main process.

Reply:
if we're OK accepting this error we should just not insert the empty string to begin with, imho.
trying to remove it afterwards feels like a bit of a hack.

Reply:
Agree. Updated.